vmdb/app/models/ems_kubernetes.rb

```diff
@@ -1,8 +1,4 @@
 class EmsKubernetes < EmsContainer
-  has_many :container_nodes,                      :foreign_key => :ems_id, :dependent => :destroy
```

Comment:
> That is part of the patch:
> 
> containers: relationships belong to EmsContainer simon3z@c7f35c8
> 
> and the relationships are just moved to EmsContainer because they're not specific to Kubernetes.

they are going to move to the container mixin as part of openshift provider patch.
potentially, amazon container provider will inherit from ems_container, and not all entities will be relevant to it.